frontend/src/Store/Actions/trackFileActions.js

```diff
@@ -20,21 +24,66 @@ export const section = 'trackFiles';
 export const defaultState = {
   isFetching: false,
   isPopulated: false,
+  sortKey: 'path',
+  sortDirection: sortDirections.ASCENDING,
+
   error: null,
   isDeleting: false,
   deleteError: null,
   isSaving: false,
   saveError: null,
-  items: []
+  itemsAreMapped: true,
+  items: [],
+
+  columns: [
+    {
+      name: 'path',
+      label: 'Path',
+      isSortable: true,
+      isVisible: true,
+      isModifiable: false
+    },
+    {
+      name: 'size',
+      label: 'Size',
+      isSortable: true,
+      isVisible: true
+    },
+    {
+      name: 'dateAdded',
+      label: 'Date Added',
+      isSortable: true,
+      isVisible: true
+    },
+    {
+      name: 'quality',
+      label: 'Quality',
+      isSortable: false,
```

Comment:
Should we make this sortable?

Reply:
How would we define the ordering?  These aren't linked up to a quality profile so not sure we have anything existing to go from?

Reply:
Ah that's true, would have to use either default quality weight or alphabetical... But I doubt we have default weight available do we?

Reply:
I'm not sure... I've never looked closely enough at the quality code. Happy to have a go if there something we can use.

Reply:
Everything else looks and works well from what I can see. If you want to try this the best place to look is manual import where its already implemented. Basically you will need to add a sortPredicate to the store for quality like so

https://github.com/lidarr/Lidarr/blob/c4a067751ab0d69b2dd6fccce86ac47ee36777bd/frontend/src/Store/Actions/interactiveImportActions.js#L49-L51

and then add and pass the quality weight from the api similar to whats done here to supplement the resource

https://github.com/lidarr/Lidarr/blob/c4a067751ab0d69b2dd6fccce86ac47ee36777bd/src/Lidarr.Api.V1/ManualImport/ManualImportModule.cs#L53-L63

Reply:
Added sorting by quality (and fixed the broken sorting in the interactive import modal - it was sorting by undefined not the quality weight)